core/src/main/java/io/kestra/core/models/flows/State.java

```diff
@@ -269,6 +269,7 @@ public boolean isBreakpoint() {
         public boolean isRetrying() {
             return this == Type.RETRYING || this == Type.RETRIED;
         }
+        public boolean isRestarted() { return  this == Type.RESTARTED; }
```

Comment:
```suggestion
        public boolean isRestarted() { 
            return  this == Type.RESTARTED;
        }
```

Please respect our coding format rules